tests/util/common.sh

```
@@ -285,6 +352,8 @@ install_operator() {
         local versions_output
         versions_output=$(runme run "${prefix}:check-packagemanifest-versions" 2>/dev/null || echo "")
 
+        echo $versions_output
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)